EXAMPLES.md

```diff
@@ -106,13 +106,15 @@ export default handleAuth({
         custom_param: 'custom'
       },
       returnTo: '/custom-page'
-    });
+    })(req,res);
```

Comment:
This isn't necessary

Reply:
It didn't run on my Next.js project without calling the function with these parameters, are you 100% sure this isn't necessary ? I've got stalled requests (the function didn't run) when using this without the `(req,res)`

Reply:
> This isn't necessary

It didn't run on my Next.js project without calling the function with these parameters, are you 100% sure this isn't necessary ? I've got stalled requests (the function didn't run) when using this without the `(req,res)`

Reply:
@jbednar97 

> are you 100% sure this isn't necessary ?
Yes, I've just confirmed this in the sample app https://github.com/auth0/nextjs-auth0/compare/main...test-handle-login

Could you share a repo that demonstrates your issue?

Reply:
> @jbednar97
> 
> > are you 100% sure this isn't necessary ?
> 
> Yes, I've just confirmed this in the sample app [main...test-handle-login](https://github.com/auth0/nextjs-auth0/compare/main...test-handle-login)
> 
> Could you share a repo that demonstrates your issue?

It was in a private repo, I'll need to re-create it later tonight if you're willing to wait, I cannot really share the current repo. In the example, you aren't using `pageRouterAuth`, which might indeed fire the function for you, but it doesn't if you use the example code, and navigate to it with a link with href `/api/auth/invite` or something similar

Reply:
@jbednar97 - `pageRouterAuth.handleLogin` is the same as `handleLogin` - happy to wait for you to share your example

Reply:
@adamjmcgrath alrighty then, will get back to you, thank you